config.template.js

```diff
@@ -19,7 +19,7 @@ const G3W_KEYS = {
 let conf = {
   assetsFolder:  './src/assets',                                            // path to G3W-CLIENT assets folder
   pluginsFolder: './src/plugins',                                           // path to G3W-CLIENT plugins folder
-  admin_plugins_folder:   '../g3w-admin/g3w-admin',                         // path to G3W-ADMIN plugins folder
+  admin_folder:   '../g3w-admin/g3w-admin',                                 // path to G3W-ADMIN code folder
```

Comment:
I named it `admin_plugins_folder` because that is the path relative to the admin packages folder (aka. django apps) and not just the path to [`g3-admin`](https://github.com/g3w-suite/g3w-admin) source code (ie. `../g3w-admin/`)

Reply:
@Raruto ok .. so i revert the changes?

Reply:
@Raruto Do i rever commit or rename admin_folder to admin_apps_folders?

Reply:
@volterra79 for now I would stick on "`admin_plugins_folder`":

```js
  pluginsFolder:          './src/plugins',            // path to G3W-CLIENT plugins folder
  admin_plugins_folder:   '../g3w-admin/g3w-admin',   // path to G3W-ADMIN plugins folder
```

Reply:
@Raruto ok.